cmd/nerdctl/container_run_windows_test.go

```diff
@@ -108,3 +108,20 @@ func TestRunProcessContainer(t *testing.T) {
 
 	assert.Assert(t, !strings.Contains(inspectOutput, "hyperv"))
 }
+
+// Note that the current implementation of this test is not ideal, since it relies on internal HCS details that
+// Microsoft could decide to change in the future (breaking both this unit test and the one in containerd itself):
+// https://github.com/containerd/containerd/pull/6618#discussion_r823302852
+func TestRunProcessContainerWithDevice(t *testing.T) {
+	testutil.DockerIncompatible(t)
```

Comment:
Why incompatible?

Reply:
I didn't notice that line earlier, but now I look, every test in this file appear to be `DockerIncompatible`, and some appear to be needlessly-so (like this one and `TestRunProcessIsolated`, which are both trivially "run container with Docker-compatible command-line, and check stdout" tests). Is there something in the underlying test suite that makes running against a WIndows Docker build inherently fail?

`container_run_user_windows_test.go` doesn't seem to have this limitation though, so perhaps this is a long-ago copy-and-paste oversight that has kept propagating from a test which did use Docker-incompatible features?

Reply:
Yeah, I wasn't 100% sure if that line was actually necessary. I included it because it was present in `TestRunProcessIsolated`, despite the absence of any Docker-incompatible flags. I figured that the same compatibility constraints apply to both tests, but if that line is erroneous in the existing test then I've effectively just duplicated that mistake.